bazel-mode.el

```diff
@@ -21,9 +21,20 @@
 ;;
 ;;; Code:
 
+(defconst bazel-mode-syntax-table
+  (let ((table (make-syntax-table)))
+	; single and multiple line comment start
+	(modify-syntax-entry ?# "< 14" table)
+	; single line comment end
+	(modify-syntax-entry ?\n ">" table)
+	table)
+  "Syntax table for `bazel-mode'.")
+
 ;;;###autoload
 (define-derived-mode bazel-mode prog-mode "Bazel"
-  "Major mode for editing Bazel BUILD and WORKSPACE files.")
+  "Major mode for editing Bazel BUILD and WORKSPACE files."
+  :syntax-table bazel-mode-syntax-table
```

Comment:
This isn't necessary, Emacs will automatically use the right variable by default, see https://www.gnu.org/software/emacs/manual/html_node/elisp/Derived-Modes.html.

Reply:
It seems I can get rid of this line but not the following one.